framework/transport/ieee1905_transport/ieee1905_transport_packet_processing.cpp

```diff
@@ -541,18 +541,18 @@ std::ostream &Ieee1905Transport::Packet::print(std::ostream &os) const
     std::stringstream ss;
 
     ss << "ethernet header:" << std::endl;
-    ss << "  dst        : " << std::hex << std::setfill('0') << std::setw(2) << (unsigned)dst[0]
-       << ":" << std::hex << std::setfill('0') << std::setw(2) << (unsigned)dst[1] << ":"
-       << std::hex << std::setfill('0') << std::setw(2) << (unsigned)dst[2] << ":" << std::hex
-       << std::setfill('0') << std::setw(2) << (unsigned)dst[3] << ":" << std::hex
-       << std::setfill('0') << std::setw(2) << (unsigned)dst[4] << ":" << std::hex
-       << std::setfill('0') << std::setw(2) << (unsigned)dst[5] << std::endl;
-    ss << "  src        : " << std::hex << std::setfill('0') << std::setw(2) << (unsigned)src[0]
-       << ":" << std::hex << std::setfill('0') << std::setw(2) << (unsigned)src[1] << ":"
-       << std::hex << std::setfill('0') << std::setw(2) << (unsigned)src[2] << ":" << std::hex
-       << std::setfill('0') << std::setw(2) << (unsigned)src[3] << ":" << std::hex
-       << std::setfill('0') << std::setw(2) << (unsigned)src[4] << ":" << std::hex
-       << std::setfill('0') << std::setw(2) << (unsigned)src[5] << std::endl;
+    ss << "  dst        : " << std::hex << std::setfill('0') << std::setw(2) << (unsigned)dst.oct[0]
+       << ":" << std::hex << std::setfill('0') << std::setw(2) << (unsigned)dst.oct[1] << ":"
+       << std::hex << std::setfill('0') << std::setw(2) << (unsigned)dst.oct[2] << ":" << std::hex
+       << std::setfill('0') << std::setw(2) << (unsigned)dst.oct[3] << ":" << std::hex
+       << std::setfill('0') << std::setw(2) << (unsigned)dst.oct[4] << ":" << std::hex
+       << std::setfill('0') << std::setw(2) << (unsigned)dst.oct[5] << std::endl;
+    ss << "  src        : " << std::hex << std::setfill('0') << std::setw(2) << (unsigned)src.oct[0]
+       << ":" << std::hex << std::setfill('0') << std::setw(2) << (unsigned)src.oct[1] << ":"
+       << std::hex << std::setfill('0') << std::setw(2) << (unsigned)src.oct[2] << ":" << std::hex
+       << std::setfill('0') << std::setw(2) << (unsigned)src.oct[3] << ":" << std::hex
+       << std::setfill('0') << std::setw(2) << (unsigned)src.oct[4] << ":" << std::hex
+       << std::setfill('0') << std::setw(2) << (unsigned)src.oct[5] << std::endl;
```

Comment:
You don't need to do that. We already have operator overload for this. If you do ` << dst` it will be converted to a string.

Reply:
right, fixed.